cmd/kubeadm/app/preflight/checks_darwin.go

```diff
@@ -0,0 +1,24 @@
+// +build darwin
```

Comment:
Might want to add a comment about mac below, my brain kind of SEGV'd when reading this at first.

Reply:
do you mean adding a note that `darwin == macos` in golang?

Reply:
Do we actually need the conditional here for a mac? Do we expect users to run `kubeadm` on a mac?

Reply:
we have a typecheck from hack that will fail if the symbol is not defined for darwin.

Reply:
I think that's the #1 development platform, so we should probably do right by the developers.

Reply:
@neolit123 - just a comment below is fine. 
// This is a macos stub, yadda yadda.